src/main/java/com/stripe/model/SubscriptionSchedule.java

```diff
@@ -467,6 +467,38 @@ public SubscriptionSchedule release(
         ApiResource.RequestMethod.POST, url, params, SubscriptionSchedule.class, options);
   }
 
+  @Getter
+  @Setter
+  @EqualsAndHashCode(callSuper = false)
+  public static class AddInvoiceItemsEntry extends StripeObject {
```

Comment:
This class shouldn't be here I think and instead should be inside `Subscription` which is the core resource. You will need to change the `client_metadata`. Also the name should be `AddInvoiceItem` with no S or extra word

Reply:
So the name is still incorrect, it shouldn't be `AddInvoiceItemsEntry` it should be `AddInvoiceItem`